src/epggrab/otamux.c

```diff
@@ -492,6 +492,7 @@ epggrab_ota_kick_cb ( void *p )
     [MM_EPG_ONLY_PSIP]               = "psip",
     [MM_EPG_ONLY_UK_FREESAT]         = "uk_freesat",
     [MM_EPG_ONLY_UK_FREEVIEW]        = "uk_freeview",
+    [MM_EPG_ONLY_UK_CABLE]           = "uk_cable",
```

Comment:
It seems like Virgin only specific thing, so I would probably prefer uk_cable_virgin name.